pkg/dashboard/handlers/kubeHandlers.go

```diff
@@ -69,3 +70,13 @@ func (h *KubeHandler) Describe(c *gin.Context) {
 
 	c.String(http.StatusOK, res)
 }
+
+func (h *KubeHandler) GetNameSpaces(c *gin.Context) {
+	res, err := h.Data.GetNameSpaces()
+	if err != nil {
+		_ = c.AbortWithError(http.StatusBadRequest, err)
+		return
+	}
+
+	c.String(http.StatusOK, res)
```

Comment:
I'd rather return JSON, as it's a more structured and extensible format.

Reply:
updated